### PR TITLE
add console.log

### PR DESCRIPTION
**Pull request by clarktsc39**

### CODE REVIEW

**Comment by clarktsc39** [comment]
completeness: 3

Comfort: 2

**Comment by clarktsc39** [comment]
Completeness: 4

Comfort: 3

I had a little trouble with if else statements. Not sure how they work... Also, took me a lot more time to plan out my story and write the if and if else statements so that the story would flow the way I envisioned.

File: js-calculator/calculator.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+var calc = {
```
Comment (amaseda) [inline js-calculator/calculator.js:1]: Looks like you still left your calculator assignment in this repo!

File: starter/script.js [hunk 2]
```diff
@@ -1 +1,35 @@
-// put code here!
+var buffer = function(){
```
Comment (amaseda) [inline starter/script.js:1]: Nice! Dig that you put repeated code into a function.

File: starter/script.js [hunk 3]
```diff
@@ -1 +1,35 @@
...
+
+console.log("You want to travel to a deserted island...")
+buffer()
+var pathInput = prompt("You can either take a plane or a helicopter..(Please enter 'p' or 'h')")
+
+if(pathInput == "p"){
+  console.log("You take the plane and everything is going well, in fact - you're ahead of scuedule. You hit some very bad turbulance and the plane crashes; you hit your head during the crash and it's all bloody now.")
+  buffer()
+  var responsePlaneCrash = prompt("You wounder what you want to do ... and decide to\n('b') - bandage yourself up with the left over medical supplies you could find in the cock pit.\n('w') - take a rest and wait until you feel better\n('d') - do nothing.")
+  if(responsePlaneCrash== 'b'){
```
Comment (amaseda) [inline starter/script.js:13]: This is a stylistic thing, but be consistent with your spacing. It'll make your code more readable and more pleasing on the eyes (which'll be important when employers look through your code!).

File: starter/script.js [hunk 4]
```diff
@@ -1 +1,35 @@
...
+    }
+  }
+  else if(responseToRabidMan == 'w'){
+    console.log("You thing the best thing to do is rest and wait to start feeling better?'")
+    buffer()
+    var WaitingAround = prompt("Still a little out of it from the crash, you bandage up your head as well as you can but you start feeling woosy....\n(s) - .. All the blood rushes away from your head, starving you of oxygen and you die quickly in a slow manner\n('f') - you fight through the woozy feeling start crawling and slowly work your way to you feet safely.")
+    if(WaitingAround == 's'){
+      console.log("You're dead!")
+      buffer()
+
```
Comment (amaseda) [inline starter/script.js:35]: You're off to a great start! You didn't finish your code here but, from what I can tell, you've got a good grasp on if-else statements. Happy to go over them with you this week.

File: starter/script.js [hunk 5]
```diff
@@ -1 +1,35 @@
...
+        var rabidManResponse = prompt("That's no where near how far it is")
+      }
+      console.log("The rabid dude shreeks, 'Wow " + rabidManResponse + " you must have been studying this place for years!' You tell him to wait there while you go to the bathroom in the woods (he's stupid). You continue on your journey and finally get to....Burger King, isn't that place golden?'" )
+      buffer()
+    }
+  }
+  else if(responseToRabidMan == 'w'){
+    console.log("You thing the best thing to do is rest and wait to start feeling better?'")
+    buffer()
+    var WaitingAround = prompt("Still a little out of it from the crash, you bandage up your head as well as you can but you start feeling woosy....\n(s) - .. All the blood rushes away from your head, starving you of oxygen and you die quickly in a slow manner\n('f') - you fight through the woozy feeling start crawling and slowly work your way to you feet safely.")
```
Comment (amaseda) [inline starter/script.js:31]: Another style thing: don't capitalize variable names. Javascript convention is camel case (e.g., `var waitingAround = ...`)

**Comment by amaseda** [comment]
:+1:

**Pull request closed without merging**